tools/nuget/validate_package.py

```diff
@@ -276,9 +294,11 @@ def validate_nuget(args):
     full_nuget_path = os.path.join(args.package_path, nuget_file_name)
 
     if "Gpu" in nuget_file_name:
-        is_gpu_package = True
+        is_gpu_package = bool("Gpu-" not in nuget_file_name)
+        is_gpu_dependent_package = bool("Gpu-" in nuget_file_name)
```

Comment:
```suggestion
        is_gpu_package = nuget_file_name.endswith('Gpu')
        is_gpu_dependent_package = not is_gpu_package
```

Reply:
We couldn't use endswith because the package name includes the version string.